docs/docusaurus/src/theme/NavbarItem/DropdownNavbarItem.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import DropdownNavbarItem from '@theme-original/NavbarItem/DropdownNavbarItem';
+
+export default function DropdownNavbarItemWrapper(props) {
+  return (
+    <DropdownNavbarItem {...props} href={'#'} />
```

Comment:
You can see the current behavior in [https://docs.greatexpectations.io/docs/home/]() by clicking the label of the dropdown (not one of the options but the label itself). It makes a weird redirect to why gx cloud page, so here I'm just overriding that behavior so that it stays on the current page. I think this it the simplest solution without having to change the actual implementation of the dropdown from docusaurus, as we're trying to keep that need as a minimum because it's not recommended by the documentation

Reply:
I agree with @tyler-hoffman , what is the intent behavior if this `href={"#"}`  I'm not familiarized with this approach.

Reply:
@josectobar an href of just `#` means it's a url fragment. When it's not followed by an element id, it's a reference to the top of the page. So this basically hijacks the href to point to the current page, rather than the "why gx cloud" page that it otherwise defaults to. I did a tiny bit of looking to figure out why docusaurus defaults to that, and reported some intermediate findings here: https://greatexpectationslabs.slack.com/archives/C03B8DZCJ07/p1708537374187499.

Reply:
I'm not able to see the message as I'm not part of the org :eyes: could you paste that info here?

Reply:
I'd like to see us figure out a better solution, but I'm okay with this in the short term!